<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="enabled">
            <bool>true</bool>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>694</width>
                <height>451</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="windowTitle">
            <string>Book Rental</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QTableView" name="rented_books_QTableView">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>10</y>
                        <width>491</width>
                        <height>391</height>
                    </rect>
                </property>
            </widget>
            <widget class="QGroupBox" name="groupBox">
                <property name="geometry">
                    <rect>
                        <x>510</x>
                        <y>10</y>
                        <width>171</width>
                        <height>80</height>
                    </rect>
                </property>
                <property name="title">
                    <string>Search</string>
                </property>
                <widget class="QComboBox" name="search_comboBox">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>50</y>
                            <width>151</width>
                            <height>22</height>
                        </rect>
                    </property>
                </widget>
                <widget class="QLineEdit" name="search_input">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>20</y>
                            <width>151</width>
                            <height>20</height>
                        </rect>
                    </property>
                </widget>
            </widget>
            <widget class="QGroupBox" name="groupBox_2">
                <property name="geometry">
                    <rect>
                        <x>510</x>
                        <y>140</y>
                        <width>171</width>
                        <height>51</height>
                    </rect>
                </property>
                <property name="title">
                    <string>Books</string>
                </property>
                <widget class="QPushButton" name="btn_show_books">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>20</y>
                            <width>75</width>
                            <height>23</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>Show All</string>
                    </property>
                </widget>
            </widget>
            <widget class="QGroupBox" name="groupBox_3">
                <property name="geometry">
                    <rect>
                        <x>510</x>
                        <y>190</y>
                        <width>171</width>
                        <height>51</height>
                    </rect>
                </property>
                <property name="title">
                    <string>Renters</string>
                </property>
                <widget class="QPushButton" name="btn_show_renters">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>20</y>
                            <width>75</width>
                            <height>23</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>Show All</string>
                    </property>
                </widget>
            </widget>
            <widget class="QGroupBox" name="groupBox_4">
                <property name="geometry">
                    <rect>
                        <x>510</x>
                        <y>240</y>
                        <width>171</width>
                        <height>161</height>
                    </rect>
                </property>
                <property name="title">
                    <string>Statistics</string>
                </property>
                <widget class="QLabel" name="label">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>20</y>
                            <width>128</width>
                            <height>13</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>Number of overdue books:</string>
                    </property>
                </widget>
                <widget class="QLabel" name="overdue_books_count">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>40</y>
                            <width>41</width>
                            <height>16</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>000</string>
                    </property>
                </widget>
                <widget class="QLabel" name="rented_books_count">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>80</y>
                            <width>18</width>
                            <height>13</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>000</string>
                    </property>
                </widget>
                <widget class="QLabel" name="label_7">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>60</y>
                            <width>121</width>
                            <height>13</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>Number of rented books:</string>
                    </property>
                </widget>
            </widget>
            <widget class="QGroupBox" name="groupBox_5">
                <property name="geometry">
                    <rect>
                        <x>510</x>
                        <y>90</y>
                        <width>171</width>
                        <height>51</height>
                    </rect>
                </property>
                <property name="title">
                    <string>Rents</string>
                </property>
                <widget class="QPushButton" name="add_new_book_rent_button">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>20</y>
                            <width>75</width>
                            <height>23</height>
                        </rect>
                    </property>
                    <property name="text">
                        <string>Add New</string>
                    </property>
                </widget>
            </widget>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>694</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
